### DISCUSSION TITLE
Could I use Quorum as Layer 2 solution, particular ZK rollup?

### DISCUSSION THREAD

**Discussion by SuperHeroX** [ConsenSys/quorum#1378] [Q&A]
I've read previous Q&A and found we could use Quorum as a sidechain with 0 gas. (We could use POA to keep the network permissioned to prevent attacking). I'd like to ask whether Quorum supports other scaling solutions like ZK rollup? Are there any tutorials?

**Comment by baptiste-b-pegasys** [comment]
Hello, I think it would be logically feasible as they exist on eth. If you can find a tutorial on how to run a layer 2 on ethereum, you would be able to run on your own GoQuorum net/chain.
But I fear it depends on the layer 2 implementation, so I would look at an open source layer 2 if there is any tutorial available.
Like this one maybe : https://github.com/AstarNetwork/ZKRollups

Edit: we have sumo for implementing rollups: https://github.com/ConsenSys/sumo

**Reply by earlyciada** [reply]
Hello, I am not able to access "https://github.com/ConsenSys/sumo", 404 error returned.
